scripts/shiptemplates/OLD.lua

```diff
@@ -387,7 +387,8 @@ template:setJumpDrive(true)
 template:setDockClasses(_("class", "Starfighter"), _("class", "Frigate"), _("class", "Corvette"))
 template:setSharesEnergyWithDocked(true)
 
-variation = template:copy("Ender"):setType("playership"):setLocaleName(_("playerShip", "Ender"))
+variation = template:copy("Ender")--:setType("playership"):setLocaleName(_("playerShip", "Ender"))
```

Comment:
you removed the playership type here?

Reply:
ah, dang! that was just meant as a temporary test to easyly compare both ships in the database, in preparation to write the description for the ender. Will fix it